tests/helpers/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Page, expect } from "@playwright/test";
+
+export const selectFacultyAndRegistration = async (page: Page) => {
+  console.log("selectFacultyAndRegistration - 1");
+  await page.waitForTimeout(1000);
+  const facultyButton = page.getByText(/wybierz swój/i);
+  await facultyButton.click();
+  console.log("selectFacultyAndRegistration - 2");
+
+  // Wait for the dropdown content to be visible
+  const selector = await page.waitForSelector('[role="option"]', {
+    state: "visible",
+  });
+  console.log("selectFacultyAndRegistration - 2.2", selector);
+  // Wait specifically for the W5 option to be available
+  const chosenFaculty = page
+    .locator('[role="option"]')
+    .filter({ hasText: "W5" });
+  await expect(chosenFaculty).toBeVisible();
+  console.log("selectFacultyAndRegistration - 2.5");
+  await chosenFaculty.click();
+  console.log("selectFacultyAndRegistration - 3");
+  const registrationButton = page.locator('button[name="registration"]');
```

Comment:
https://playwright.dev/docs/locators

tutaj masz listę locatorów, których wypada używać - page.locator to jest escape hatch i raczej używa się go do scrapowania stronek a nie pisania testów